### DISCUSSION TITLE
NZXT HUB

### DISCUSSION THREAD

**Discussion by miklas206** [Rem0o/FanControl.Releases#660] [Q&A]
Hey i just wondered if anyone have tried to make a plugin for the NZXT rgb and fan hub would be nice :)

**Comment by louiswutsch** [comment]
This issue was posted recently in the LibreHardwareMonitor repo, and I am guessing the issue would need resolving in that repo rather than in FanControl: 
https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/702#issue-1212919954
